docs/source/firstDraft.py

```diff
@@ -1,17 +1,56 @@
-from qgis.core import QgsProject
```

Comment:
Skript jste umistil do adresare `docs`, kam patri dokumentace. To neni uplne nejvhodnejsi misto.

Reply:
presunuto do domovskeho adresare `2024-a-qgis-ruian-plugin`